pkg/operator/condition/condition.go

```diff
@@ -0,0 +1,56 @@
+package condition
+
+const (
+	// ManagementStateDegradedConditionType represents a condition when the operator management state is changed from Managed to other value.
+	// Possible reasons are Unmanaged, Removed or Unknown. Any of these cases means the operator is not actively managing the operand.
+	ManagementStateDegradedConditionType = "ManagementStateDegraded"
+
+	// UnsupportedConfigOverridesUpgradeableConditionType represents a condition when operator unsupported config overrides is changed.
```

Comment:
is true if UnsupportedConfigOverrides is set.

Reply:
it applies to all, I will replace